### PR TITLE
COMP: unused variable 'numberOfPoints'

### PR DESCRIPTION
**Pull request by hjmjohnson**
A variable that is not used is removed.

### CODE REVIEW

File: src/itkSWCMeshIO.cxx [hunk 1]
```diff
@@ -236,7 +236,6 @@ ::ReadCells(void * buffer)
   auto * data = static_cast<unsigned int *>(buffer);
 
   // Skip root
-  const SizeValueType numberOfPoints = this->GetNumberOfPoints();
   SizeValueType cellBufferIndex = 0;
   for (SizeValueType pointIndex = 1; pointIndex < this->GetNumberOfPoints(); ++pointIndex)
```
Comment (dzenanz) [inline src/itkSWCMeshIO.cxx:240]: That variable was probably meant to be used here.

**Review by dzenanz — APPROVED** [review]

**Comment by jhlegarreta** [comment]
See if PR #5 helps with the CIs.

**Comment by dzenanz** [comment]
Please rebase

**Review by thewtex — APPROVED** [review]
@hjmjohnson thanks!

**Pull request merged**